index.html

```diff
@@ -164,18 +171,23 @@ <h2>Multi-Post Stories</h2>
         </div>
       </div>
     </section>
-    <section>
+
+
+
+
+
+    <section class="about-sect">
       <div class="about-me">
         <img class="about-me-img" src="./images/illustration-2-about-me-mobile.svg" alt="" />
         <h3>About Me</h3>
         <p class="description">
           I can help you build a product , feature or website Look through some of
           my work and experience! If you like what you see and have a project you
-          need coded, don’t hestiate to contact me.
+          need coded, don't hestiate to contact me.
         </p>
-        <p class="resume-btn">Get my Resume</p>
+        <button class="resume-btn" type="button">Get my Resume</button>
       </div>
-      <div>
```

Comment:
Please use the same background images from the design. Kindly see the screenshot below for reference.
![about-me](https://user-images.githubusercontent.com/35504149/131216479-30645b92-974c-4541-8b1d-c2ebb7360e1c.png)